README_zh.md

```diff
@@ -4,162 +4,245 @@
+      <li><a href="#超链接值">超链接值</a></li>
       <li>
-        <a href="#formula-value">公式值</a>
+        <a href="#公式值">公式值</a>
         <ul>
-          <li><a href="#shared-formula">共享公式</a></li>
-          <li><a href="#formula-type">公式类型</a></li>
+          <li><a href="#共享公式">共享公式</a></li>
+          <li><a href="#公式类型">公式类型</a></li>
+          <li><a href="#数组公式">数组公式</a></li>
         </ul>
       </li>
-      <li><a href="#rich-text-value">富文本值</a></li>
-      <li><a href="#boolean-value">布尔值</a></li>
-      <li><a href="#error-value">错误值</a></li>
+      <li><a href="#富文本值">富文本值</a></li>
+      <li><a href="#布尔值">布尔值</a></li>
+      <li><a href="#错误值">错误值</a></li>
     </ul>
   </li>
-  <li><a href="#config">配置</a></li>
-  <li><a href="#known-issues">已知的问题</a></li>
-  <li><a href="#release-history">发布历史</a></li>
+  <li><a href="#配置">配置</a></li>
+  <li><a href="#已知的问题">已知的问题</a></li>
+  <li><a href="#发布历史">发布历史</a></li>
 </ul>
 
-## <a id="importing">导入</a>
 
-默认导出是带有 Promise polyfill 转换的 ES5 版本 - 因为这会提供最高的兼容性。
+
+# 导入
+
+```javascript
+const ExcelJS = require('exceljs');
+```
+
+## ES5 导入
+
+要使用 ES5 编译代码，请使用 *dist/es5* 路径。
 
 ```javascript
-var Excel = require('exceljs');
-import Excel from 'exceljs';
+const ExcelJS = require('exceljs/dist/es5');
 ```
 
-但是，如果您在现代node.js版本（>=8）上使用此库，或者在使用bundler的前端（或者只关注常绿浏览器）上使用此库，我们建议使用这些导入：
+**注意：**ES5 版本对许多 polyfill 都具有隐式依赖，而 exceljs 不再明确添加。
+您需要在依赖项中添加 `core-js` 和 `regenerator-runtime`，并在导入 `exceljs` 之前在代码中包含以下引用：
 
 ```javascript
-const Excel = require('exceljs/modern.nodejs');
-import Excel from 'exceljs/modern.browser';
+// exceljs 所需的 polyfills
+require('core-js/modules/es.promise');
+require('core-js/modules/es.object.assign');
+require('core-js/modules/es.object.keys');
+require('regenerator-runtime/runtime');
+
+const ExcelJS = require('exceljs/dist/es5');
 ```
 
-# <a id="interface">接口</a>
+对于 IE 11，您还需要一个 polyfill 以支持 unicode regex 模式。 例如，
+
+```js
+const rewritePattern = require('regexpu-core');
+const {generateRegexpuOptions} = require('@babel/helper-create-regexp-features-plugin/lib/util');
 
-## <a id="create-a-workbook">创建工作簿</a>
+const {RegExp} = global;
+try {
+  new RegExp('a', 'u');
+} catch (err) {
+  global.RegExp = function(pattern, flags) {
+    if (flags && flags.includes('u')) {
+      return new RegExp(rewritePattern(pattern, flags, generateRegexpuOptions({flags, pattern})));
+    }
+    return new RegExp(pattern, flags);
+  };
+  global.RegExp.prototype = RegExp;
+}
+```
+
+## 浏览器端
+
+ExcelJS 在 *dist/* 文件夹内发布了两个支持浏览器的包：
+
+一个是隐式依赖 `core-js` polyfills 的...
+```html
+<script src="https://cdnjs.cloudflare.com/ajax/libs/babel-polyfill/6.26.0/polyfill.js"></script>
+<script src="exceljs.js"></script>
+```
+
+另一个则没有...
+```html
+<script src="--your-project's-pollyfills-here--"></script>
+<script src="exceljs.bare.js"></script>
+```
+
+
+# 接口
+
+## 创建工作簿
 
 ```javascript
 var workbook = new Excel.Workbook();
 ```
 
-## <a id="set-workbook-properties">设置工作簿属性</a>
+## 设置工作簿属性
```

Comment:
@wang1212 Removing the `a` tag will cause the anchor to fail. Is your local verification successful?

Reply:
Yes, I don’t need the `a` tag to locate the anchor point locally. Moreover, you can see that it is normal in my git repository [wang1212/exceljs](https://github.com/wang1212/exceljs/blob/master/README_zh.md)